tasks/review.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0"?>
+
+<!--
+  @file review.xml
+  Configuration for code reviews.
+
+  This file is included in the build.dist.xml template to provide the default
+  configuration for three code reviews:
+    * The PHP interpreter's built-in linter
+    * PHP Mess Detector (https://phpmd.org/)
+    * PHP Code Sniffer (https://github.com/squizlabs/PHP_CodeSniffer)
+      * Coding standard from drupal/coder (https://www.drupal.org/project/coder)
+
+  Copyright 2018 Palantir.net, Inc.
+  -->
+
+<project name="review" default="review-properties">
+
+
+    <!--
+      PHP Mess Detector configuration.
+      These values are used in the build.dist.xml template:
+        phpmd ${phpmd.directories} ${phpmd.format} ${phpmd.rulesets} -.-suffixes=${phpmd.suffixes}
+      -->
+    <property name="phpmd.directories" value="${drupal.root}/modules/custom/,${drupal.root}/themes/custom/" />
+    <property name="phpmd.format" value="text" />
+    <property name="phpmd.rulesets" value="${build.thebuild.dir}/conf/phpmd.xml" />
+    <property name="phpmd.suffixes" value="php,inc,module,install,profile" />
+
+
+    <!--
+      PHP Code Sniffer configuration.
+      These values are used in the build.dist.xml template:
+        phpcs -.-standard=${phpcs.standard} -.-ignore=${phpcs.ignore} ${phpcs.directories}
+      -->
+    <property name="phpcs.standard" value="${build.dir}/vendor/drupal/coder/coder_sniffer/Drupal/ruleset.xml" />
+    <property name="phpcs.directories" value="${drupal.root}/modules/custom ${drupal.root}/themes/custom" />
```

Comment:
Should profiles/custom be included by default, since we are linting them by default?